test/test-markdown-external.js

```diff
@@ -0,0 +1,19 @@
+
+
+Reveal.addEventListener( 'ready', function() {
+
+	QUnit.module( 'Markdown' );
+
+	test( 'Vertical separator', function() {
+		strictEqual( document.querySelectorAll( '.reveal .slides>section>section' ).length, 2, 'found two slides' );
```

Comment:
Looks like this is giving a false positive. The md file parses into a presentation where "Slide 1.1", the code sample and "Slide 1.2" are all on the same slide. The selector still returns a length of 2 since "Slide 2", which is intended to be a horizontal slide, actually becomes a vertical slide.